msal/application.py

```diff
@@ -509,6 +512,9 @@ def __init__(
                         AttributeError,  # Cache created by a different version of MSAL
                         ):
                     persisted_http_cache = {}  # Recover by starting afresh
+                except:  # Unexpected exceptions
+                    logging.exception("You may want to debug this")
```

Comment:
Is there any more info that we can provide about what went wrong? Or is this only for truly unexpected exceptions that we can't predict?

Reply:
The [expected exceptions](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/821/files#diff-4d32d284fa6acf43894719eff11cb366e5969e0b61593bfd00c68a4cf9c441caR509-R513) were already caught above. So, this last catch-all clause is really for unexpected situations.